VisualC/SDL/SDL.vcxproj

```diff
@@ -63,14 +63,14 @@
   <PropertyGroup Label="UserMacros" />
   <PropertyGroup>
     <_ProjectFileVersion>10.0.40219.1</_ProjectFileVersion>
-    <OutDir Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">$(SolutionDir)$(Platform)\$(Configuration)\</OutDir>
-    <IntDir Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">$(Platform)\$(Configuration)\</IntDir>
-    <OutDir Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">$(SolutionDir)$(Platform)\$(Configuration)\</OutDir>
-    <IntDir Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">$(Platform)\$(Configuration)\</IntDir>
-    <OutDir Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">$(SolutionDir)$(Platform)\$(Configuration)\</OutDir>
-    <IntDir Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">$(Platform)\$(Configuration)\</IntDir>
-    <OutDir Condition="'$(Configuration)|$(Platform)'=='Release|x64'">$(SolutionDir)$(Platform)\$(Configuration)\</OutDir>
-    <IntDir Condition="'$(Configuration)|$(Platform)'=='Release|x64'">$(Platform)\$(Configuration)\</IntDir>
+    <OutDir Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">$(SolutionDir)\SDL\$(Platform)\$(Configuration)\</OutDir>
```

Comment:
Correct me if I'm wrong, but isn't the VS default to put binaries in `$(SolutionDir)$(Platform)\$(Configuration)`?
I mean, this allows a non-SDL project to build, link, and run out-of-the box with SDL binaries.
When the SDL binaries are put in a `SDL` directory, `SDL3.dll` will not be found when starting to debug in VS.

Reply:
That is a fair point, the only real "solution" I can think to that would be extremely hacky. Which would be to just use a post build event to pop the files from `OutDir` into `IntDir` which would solve the problem in a technical sense but as I said, very very hacky